wazuh/wazuh-dashboards-reporting
# Enable notifications plugins in reporting

### Description
This pull request enables the notification plugin services.



<details><summary>Screenshot</summary>

Table

![image](https://github.com/user-attachments/assets/198012b9-784e-4ee4-bc0e-609b008abc34)

Details

![image](https://github.com/user-attachments/assets/aee6c990-11a0-4d22-ba81-1d9bcc1d6064)

Edit

![image](https://github.com/user-attachments/assets/274ddaf1-9d87-41dd-9570-35661e450054)

Create

![image](https://github.com/user-attachments/assets/f0e97e1f-4e12-482d-858e-9cf73d5e4b88)

</details> 

### Issues Resolved
- https://github.com/wazuh/wazuh-dashboard/issues/288

### Test

Preconditions

Remove the opensearch-reports-scheduler plugin and install the plugin from the wazuh-indexer-reporting repository.

Steps

1. Configuring an SMTP channel in the notification plugin https://opensearch.org/docs/latest/observing-your-data/notifications/index/#email-as-a-channel-type
2. Create a scheduled report definition
3. Try editing the text of the message and containing {{urlDefinition}} to see the url to generate the report.
4. In the table you should see that the notification is activated.
5. In the detail you should see the name of the channel in the title of the mail and the body of the mail.
6. Wait for the mail to be sent

### Check List
